src/app/favlist/page.tsx

```diff
@@ -18,9 +18,12 @@ const FavPage = () => {
     userId,
   });
 
-  const eventIds = favEvents.data?.map((event) => {
-    return event.id;
-  });
+  const eventIds = favEvents.data
+    // sort favEvents in the order of timeline
+    ?.sort((a, b) => (a.date > b.date ? 1 : -1))
+    .map((event) => {
```

Comment:
Can we use `orderBy` at `favEvents` router instead of writing our own code here?
ref: https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting#sorting

Reply:
@Lada496 That definitely sounds better! I'll add `orderBy` to the query.